--- a/doc/mpdconf.example
+++ b/doc/mpdconf.example
@@ -205,17 +205,14 @@ input {
 # autodetection for one sound card.
 #
 # An example of an ALSA output:
-#
-#audio_output {
-#	type		"alsa"
-#	name		"My ALSA Device"
-##	device		"hw:0,0"	# optional
-##	mixer_type      "hardware"	# optional
-##	mixer_device	"default"	# optional
-##	mixer_control	"PCM"		# optional
-##	mixer_index	"0"		# optional
-#}
-#
+
+audio_output {
+	type		"alsa"
+	name		"ES9023 I2S DAC"
+	device		"hw:0,0"
+	mixer_type      "software"
+}
+
 # An example of an OSS output:
 #
 #audio_output {
